devmgmtui/src/components/cloud/ContentArea.js

```diff
@@ -148,10 +148,12 @@ function renderTable(content, handleDownload) {
 }
 
 function renderNoResultsFound(query) {
+	const placeholderText = !query ? "Nothing to show here" : `No results found for "${query}"`;
```

Comment:
replace with

    query ? `No results found for "${query}"` : "Nothing to show here";

so as to avoid the unnecessary `NOT`ing of query

Reply:
Done